# Research

## Main Projects

### Protein and life evolution

- Understanding the gene-level, pathway-level and species-level evolution
- Learn the evolution of protein structure and function

### Protein Engineering and Design

- Using protein sequence and structure pretraining to assist protein engineering and design
- De novo design of enzymes with enhanced catalytic performance

### Deep learning models for protein dynamics

- Predict the conformation ensemble of a protein

### AlphaFold modification and application

- High-throughput protein structure prediction
- Highly adaptive structure prediction pipeline for vasatile applications


## Full Paper List

1. [Personalized Energy Adaptation through Reweighting Learning (PEARL) Force Field for Intrinsically Disordered Proteins](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.5c00140)

2. [Harnessing A Unified Multi-modal Sequence Modeling to unveil Protein-DNA Interdependency](https://www.biorxiv.org/content/10.1101/2025.02.26.640480v2.abstract)

3. [Entropy-driven zero-shot deep learning model selection for viral proteins](https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.7.013229)

4. [Discovery of Expression-Governing Residues in Proteins](https://www.biorxiv.org/content/10.1101/2025.01.06.631498v1.abstract)

5. [Protein engineering in the deep learning era](https://onlinelibrary.wiley.com/doi/full/10.1002/mlf2.12157)

6. [Protein Representation Learning with Sequence Information Embedding: Does it Always Lead to a Better Performance?](https://ieeexplore.ieee.org/abstract/document/10822035)

7. [Learning temperature-aware representations from millions of annotated protein sequences](https://openreview.net/forum?id=sOU2rNqo90)

8. [A conditional protein diffusion model generates artificial programmable endonuclease sequences with enhanced activity](https://www.nature.com/articles/s41421-024-00728-2)

9. [Pretrainable geometric graph neural network for antibody affinity maturation](https://www.nature.com/articles/s41467-024-51563-8)

10. [ReactZyme: A Benchmark for Enzyme-Reaction Prediction](https://proceedings.neurips.cc/paper_files/paper/2024/hash/2e68b2367d2e0bc8dd6f0ff86e07c2eb-Abstract-Datasets_and_Benchmarks_Track.html)

11. [Autoregressive Enzyme Function Prediction with Multi-scale Multi-modality Fusion](https://arxiv.org/abs/2408.06391)

12. [Simple, Efficient, and Scalable Structure-Aware Adapter Boosts Protein Language Models](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.4c00689)

13. [Precise Generation of Conformational Ensembles for Intrinsically Disordered Proteins via Fine-tuned Diffusion Models](https://www.biorxiv.org/content/10.1101/2024.05.05.592611v3.abstract)

14. [ProSST: Protein Language Modeling with Quantized Structure and Disentangled Attention](https://www.biorxiv.org/content/10.1101/2024.04.15.589672v3.abstract)

15. [Fusing Neural and Physical: Augment Protein Conformation Sampling with Tractable Simulations](https://arxiv.org/abs/2402.10433)

16. [Scaffold-Lab: Critical Evaluation and Ranking of Protein Backbone Generation Methods in A Unified Framework](https://www.biorxiv.org/content/10.1101/2024.02.10.579743v3.abstract)

17. [Harnessing Protein Language Model for Structure-Based Discovery of Highly Efficient and Robust PET Hydrolases](https://www.biorxiv.org/content/10.1101/2024.11.13.623508v1.abstract)

18. [How to select the best zero-shot model for the viral proteins?](https://www.biorxiv.org/content/10.1101/2024.10.06.616860v1.abstract)

19. [Mn2+-induced structural flexibility enhances the entire catalytic cycle and the cleavage of mismatches in prokaryotic argonaute proteins](https://pubs.rsc.org/en/content/articlehtml/2024/sc/d3sc06221j)

20. [DiffPack: A Torsional Diffusion Model for Autoregressive Protein Side-Chain Packing](https://proceedings.neurips.cc/paper_files/paper/2023/hash/96a54c09569ebbdd9ecb22f5012e6b66-Abstract-Conference.html)

21. [PDB-Struct: A Comprehensive Benchmark for Structure-based Protein Design](https://arxiv.org/abs/2312.00080)

22. [Str2Str: A Score-based Framework for Zero-shot Protein Conformation Sampling](https://arxiv.org/abs/2306.03117)

23. [Score-based Enhanced Sampling for Protein Molecular Dynamics](https://openreview.net/forum?id=NO3QwxuHv9#all)

24. [Phosphorylation Modification Force Field FB18CMAP Improving Conformation Sampling of Phosphoproteins](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.3c00112)

25. [Protein Sequence and Structure Co-Design with Equivariant Translation](https://arxiv.org/abs/2210.08761)

26. [Proteome-wide 3D structure prediction provides insights into the ancestral metabolism of ancient archaea and bacteria](https://www.nature.com/articles/s41467-022-35523-8)

27. [E3Bind: An End-to-End Equivariant Network for Protein-Ligand Docking](https://arxiv.org/abs/2210.06069)


28. [Balanced Force Field ff03CMAP Improving the Dynamics Conformation Sampling of Phosphorylation Site](https://www.mdpi.com/1422-0067/23/19/11285)

29. [Catalytic properties and biological function of a PIWI-RE nuclease from Pseudomonas stutzeri](https://link.springer.com/article/10.1186/s40643-022-00539-x)

30. [ParaFold: Paralleling AlphaFold for Large-Scale Predictions](https://dl.acm.org/doi/abs/10.1145/3503470.3503471)


31. [Mechanism of zinc ejection by disulfiram in nonstructural protein 5A](https://pubs.rsc.org/en/content/articlelanding/2021/cp/d0cp06360f/unauth)



32. [Combinatorial cell surface display system in Escherichia coli for noninvasive colorectal cancer detection](https://spj.science.org/doi/full/10.1097/JBR.0000000000000052)
    - Using gas vesicle protein to report E. coli which bind to colerectal cancer cells


